src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -237,6 +237,34 @@ protected final void verifyWithInlineConfigParser(String filePath, String... exp
         verify(parsedConfig, filePath, expected);
     }
 
+    /**
+     * Performs verification of multiple files with the given file paths using specified
+     * configuration and the array expected messages. Also performs verification of the config
+     * specified in the input file.
+     *
+     * @param filePath1 file path of first file to verify.
+     * @param filePath2 file path of second file to verify.
+     * @param expected an array of expected messages.
+     * @throws Exception if exception occurs during verification process.
+     */
+    protected final void verifyWithInlineConfigParser(String filePath1,
+                                                      String filePath2,
+                                                      String... expected)
+            throws Exception {
+        final TestInputConfiguration testInputConfiguration1 =
+                InlineConfigParser.parse(filePath1);
+        final TestInputConfiguration testInputConfiguration2 =
+                InlineConfigParser.parse(filePath2);
+        final DefaultConfiguration parsedConfig1 =
+                testInputConfiguration1.createConfiguration();
+        final DefaultConfiguration parsedConfig2 =
+                testInputConfiguration2.createConfiguration();
+        verifyViolations(parsedConfig1, filePath1, testInputConfiguration1.getViolations());
+        verifyViolations(parsedConfig2, filePath2, testInputConfiguration2.getViolations());
+        verify(parsedConfig1, filePath1, expected);
+        verify(parsedConfig2, filePath2, expected);
```

Comment:
Should be single execution of verify method on both files, with single config that is taken from first file.

Reply:
@shubh220922 , you still missing this point.
code should be:
```
combinedConfig = combine(parsedConfig1,parsedConfig1);
verifyViolations(combinedConfig, {filePath1, filePath1}, ....);
verify(combinedConfig, {filePath1, filePath1}, expected);
```

There should be single Checkstyle/Checker execution with both Checks activated.